pkg/operator/connectivitycheckcontroller/connectivity_check_controller.go

```diff
@@ -77,6 +79,22 @@ func (c *connectivityCheckController) Sync(ctx context.Context, syncContext fact
 		return nil
 	}
 
+	if len(operatorSpec.UnsupportedConfigOverrides.Raw) > 0 {
+		unsupportedConfigOverrides := make(map[string]interface{})
+		if err := yaml.Unmarshal(operatorSpec.UnsupportedConfigOverrides.Raw, &unsupportedConfigOverrides); err == nil {
```

Comment:
find the code where we read this in our target config controllers.  This can be eitehr json or yaml or something else, so it should be tolerant. You want runtime.Unstructured basically.

Reply:
I don't think it can be YAML. It's RawExtension which only embeds JSON.